src/Core/src/Handlers/DatePicker/DatePickerHandler.Android.cs

```diff
@@ -66,13 +70,16 @@ protected override void DisconnectHandler(MauiDatePicker platformView)
 
 		protected virtual DatePickerDialog CreateDatePickerDialog(int year, int month, int day)
 		{
-			var dialog = new DatePickerDialog(Context!, (o, e) =>
+			void OnDateSelectedCallback(object? obj, DatePickerDialog.DateSetEventArgs e)
```

Comment:
Same here, maybe `OnDateSelectedCallback` should be a private method instead of a local function?